admin-apis/addApiKeys.js

```diff
@@ -0,0 +1,99 @@
+import { generateKeyPairSync, sign } from "crypto";
+import { generateApiSignature } from "./utils.js";
+import { compressPublicKey } from "./utils.js";
+import { createPrivateKey } from "crypto";
+import axios from "axios";
+import * as dotenv from "dotenv";
+import fetchAppListByAccountKey from "./fetchAllApps.js";
+
+dotenv.config();
+
+const findAppById = async (appId) => {
+  const apps = await fetchAppListByAccountKey();
+  const foundApp = apps.find((app) => app.appId === appId);
+
+  return foundApp;
+};
+
+const updateAppUsingAccountKey = async () => {
+  const timestamp = Date.now().toString();
+  const updatedAppName = "MyApp-Updated";
+
+  // Find the app by ID
+  const app = await findAppById(process.env.APP_ID);
+  if (!app) {
+    throw new Error(`App with ID ${process.env.APP_ID} not found`);
+  }
+  console.log(app);
+
+  // generate a new key pair
+  const keyPair = generateKeyPairSync("ec", {
```

Comment:
Let's use https://nodejs.org/api/webcrypto.html#subtlegeneratekeyalgorithm-extractable-keyusages instead since that uses native node APIs and it's also compatible with browsers

Reply:
elliptic package is not safe for generating sensitive keys. Native implementations should always be preferred.

Reply:
done